db/rules_of_origin/roo_schemes_xi/introductory_notes/gsp_intro.md

```diff
@@ -1 +1,159 @@
-GSP text goes here
\ No newline at end of file
+### 	Note 1: General introduction
+
+1.1. This Annex sets out rules for all products, but the fact that a product is included in it does not mean that it is necessarily covered by the scheme of generalised tariff preferences (GSP). The list of products covered by the GSP, the scope of GSP preferences and the exclusions applicable to certain beneficiary countries are laid down in Regulation (EC) No 732/2008 (for the period from 1 January 2009 to 31 December 2011).
+
+1.2. This Annex lays down the conditions pursuant to Article 76 under which products shall be considered to originate in the beneficiary country concerned. There are four different types of rule, which vary according to the product:
+
+(a) through working or processing a maximum content of non-originating materials is not exceeded;
+
+(b) through working or processing the 4-digit Harmonized System heading or 6-digit Harmonized System sub-heading of the manufactured products becomes different from the 4-digit Harmonized System heading or 6-digit sub-heading respectively of the materials used;
+
+(c) a specific working and processing operation is carried out;
+
+(d) working or processing is carried out on certain wholly obtained materials.
+
+### Note 2: The structure of the list
+2.1. Columns 1 and 2 describe the product obtained. Column 1 gives the chapter number, 4-digit heading or 6-digit sub-heading number used in the Harmonized System, as appropriate. Column 2 gives the description of goods used in that system for that heading or chapter. For each entry in columns 1 and 2, subject to Note 2.4, one or more rules (“qualifying operations”) are set out in column 3. These qualifying operations concern only non-originating materials. Where, in some cases, the entry in column 1 is preceded by “ex”, this signifies that the rule in column 3 applies only to the part of that heading as described in column 2.
```

Comment:
Think there's a new line needed here

Reply:
👍 fixed, probably it renders the same, but it neater with the new line.